cli/lib/appdata.ts

```diff
@@ -129,7 +129,7 @@ export async function getAuthToken(): Promise< NonNullable< UserData[ 'authToken
 	}
 }
 
-export async function getSiteFromFolder(
+export async function getSiteFromAppData(
```

Comment:
`getSiteFromFolder` looks a bit confusing, I think it shoudl be eitehr `getSiteBYFolder` or `getSiteFromAPPDATA`  or `getSiteFromAPPDATAbyFOLDER

Reply:
`getSiteByFolder` sounds like a good option to me 👍